src/components/ComboBox/useListBox.tsx

```diff
@@ -292,6 +305,20 @@ const Container = styled.div<
     z-index: ${zIndex.OVERLAP};
   `
 })
+
+const HelpMessage = styled.p<{ themes: Theme }>`
+  ${({ themes }) => {
+    const { color, fontSize, spacingByChar } = themes
+
+    return css`
+      margin: 0 ${spacingByChar(0.5)} ${spacingByChar(0.5)};
+      padding: 0 ${spacingByChar(0.5)} ${spacingByChar(0.5)};
```

Comment:
imo: 必要な値だけ入れるだとだめかしら

```suggestion
      margin-inline: ${spacingByChar(0.5)};
      margin-block-end: ${spacingByChar(0.5)};
      padding-inline: ${spacingByChar(0.5)};
      padding-block-end: ${spacingByChar(0.5)};
```

Reply:
思想によりそうですが、

- ここでの上書きは特に影響がないはず
- margin, paddingのtopが無いことを明示できる

というメリットがあるのでこのまま行きたい派です

Reply:
margin や padding のショートハンドが問題になることはほとんどないし、imo なのでこのままで問題ないです!
（CSS よくわからない人がメンテナンスすることもないとは思うので）